### PR TITLE
docs: Add version and compatibility info

### PR DESCRIPTION
**Pull request by caitlinwheeless**
Adding info about SDK version and compatibility to the Getting Started page.

### CODE REVIEW

File: fern/pages/getting_started.mdx [hunk 1]
```diff
@@ -218,6 +218,30 @@ for task in tasks:
...
+```sh
+git clone https://github.com/HumanSignal/label-studio-sdk.git
+cd label-studio-sdk
+git checkout previous-version
+```
+
+Or you can simply modify you code to change the import stream as follows:
+
+```python
+from label_studio_sdk import Client
```
Comment (makseq) [inline fern/pages/getting_started.mdx:238]:
```suggestion
from label_studio_sdk._legacy import Client
```

File: fern/pages/getting_started.mdx [hunk 2]
```diff
@@ -218,6 +218,30 @@ for task in tasks:
...
+In June 2024, we released SDK 1.0. The previous SDK (version &lt; 1) is deprecated and no longer supported. We recommend upgrading to [the latest version](https://github.com/HumanSignal/label-studio-sdk). 
+
+If you still want to use the older version, you can install it using `pip install "label-studio-sdk<1"`. 
+
+You can also check out an older branch version in the GitHub repository:
+
+```sh
+git clone https://github.com/HumanSignal/label-studio-sdk.git
+cd label-studio-sdk
+git checkout previous-version
```
Comment (makseq) [inline fern/pages/getting_started.mdx:232]: Will this command work? Do we really have previous-version? The last one was 0.0.34, you can put it here.

**Review by makseq — CHANGES_REQUESTED** [review]

File: fern/pages/getting_started.mdx (hunk 2 above)
Comment (caitlinwheeless) [reply fern/pages/getting_started.mdx:232]: You're right, it doesn't work. I get this: `error: pathspec 'previous-version' did not match any file(s) known to git`

@niklub do you have suggestions for a fix? This command is also in the label-studio-sdk README.

Comment (makseq) [reply fern/pages/getting_started.mdx:232]: it should be this: 

```
git fetch origin
git checkout release/0.0.34
```

Comment (caitlinwheeless) [reply fern/pages/getting_started.mdx:232]: That works. 

`branch 'release/0.0.34' set up to track 'origin/release/0.0.34'.
Switched to a new branch 'release/0.0.34'`

**Pull request merged**